core/server/apps/amp/lib/helpers/amp_content.js

```diff
@@ -128,20 +128,20 @@ function getAmperizeHTML(html, post) {
     html = urlService.utils.makeAbsoluteUrls(html, urlService.utils.urlFor('home', true), post.url).html();
 
     if (!amperizeCache[post.id] || moment(new Date(amperizeCache[post.id].updated_at)).diff(new Date(post.updated_at)) < 0) {
-        return new Promise(function (resolve) {
-            amperize.parse(html, function (err, res) {
+        return new Promise((resolve) => {
+            amperize.parse(html, (err, res) => {
                 logging.info('amp.parse', post.url, moment().diff(startedAtMoment, 'ms') + 'ms');
 
                 if (err) {
                     if (err.src) {
                         logging.error(new errors.GhostError({
-                            message: 'AMP HTML couldn\'t get parsed: ' + err.src,
+                            message: `AMP HTML couldn\'t get parsed: ${err.src}`,
                             err: err,
                             context: post.url,
                             help: i18n.t('errors.apps.appWillNotBeLoaded.help')
                         }));
                     } else {
-                        logging.error(new errors.GhostError({err: err}));
+                        logging.error(new errors.GhostError({err}));
```

Comment:
?

Reply:
Looks like [paramater context matching](http://es6-features.org/#ParameterContextMatching)

Reply:
No, this is the [property shorthand](http://es6-features.org/#PropertyShorthand) feature

Reply:
Yes, it's shorthand. So is it an issue?

Reply:
No issue 👍